rules-java-archives/addon/pom.xml

```diff
@@ -15,12 +15,31 @@
 
         <!-- Local Dependencies -->
         <dependency>
+            <groupId>org.jboss.windup.maven</groupId>
+            <artifactId>nexus-indexer-data</artifactId>
+            <version>6.0</version>
+            <scope>compile</scope>
+            <optional>true</optional>
```

Comment:
Not sure what you mean by external. It's something this needs to bring along, but not the transitive dependencies, hence <optional>, I think.